stream_alert/shared/config.py

```diff
@@ -33,6 +33,24 @@ class TopLevelConfigKeys(object):
 class ConfigError(Exception):
     """Exception class for config file errors"""
 
+class SchemaSorter(object):
+    """Sort Schema by Priority"""
```

Comment:
Ideally, describe how this comparison function determines sort order. 

For example, what happens if priority is missing? What if a priority is duplicated?